Sources/Components/SelectionBoxes/Selectionbox.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+/* Selection box for selecting a singel item */
+
+public class RadioButton: Selectionbox {
```

Comment:
I think we should place RadioButton in its own file, same as Checkbox makes it easier to find the files using Quick Open.

Reply:
I only put them in the same folder so I could use the fileprivate keyword for the method these classes are suppose to override